Sources/Conduit/Networking/TaskResponse.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  TaskResponse.swift
+//  Conduit
+//
+//  Created by Eneko Alonso on 4/3/20.
+//  Copyright © 2020 MINDBODY. All rights reserved.
+//
+
+import Foundation
+
+/// Encapsulate received data, HTTP response, error, and metrics, where available
+public struct TaskResponse {
+    var data: Data?
+    var response: HTTPURLResponse?
+    var expectedContentLength: Int64?
+    var error: Error?
+    @available(iOS 10, *)
+    lazy var metrics: URLSessionTaskMetrics? = nil
```

Comment:
These should be `public`, minus the `expectedContentLength`

Reply:
Oh, good catch! Saved me from doing a patch release :)